README.md

```diff
@@ -38,29 +38,31 @@ pub fn get_value_from_js(value: JsValue) -> Result<(), JsValue> {
 Note that, even though it might often be the case, this library doesn't attempt to be strictly compatible with either [`serde_json`](https://docs.serde.rs/serde_json/) or, correspondingly, `JsValue::from_serde` / `JsValue::into_serde`, instead prioritising better compatibility with common JavaScript idioms and representations.
 
 Supported types and values for the deserialization:
- - `()` from `undefined` and `null`.
```

Comment:
What happened here with the diffs? Looks like indentation got changed - can you return as it was to keep changes smaller?